pdb/stackTrace.odin

```diff
@@ -235,7 +235,11 @@ parse_stack_trace :: proc(stackTrace: []StackFrame, sameProcess: bool, srcCodeLo
             nameBuf : [windows.MAX_PATH]u16
             pBuf := &nameBuf[0]
             nameLen := windows.GetModuleFileNameW(windows.HMODULE(stackFrame.imgBaseAddr), pBuf, len(nameBuf))
-            mi.filePath = windows.wstring_to_utf8(pBuf, cast(int)nameLen)
+            path, err := windows.wstring_to_utf8(pBuf, cast(int)nameLen)
+            if err != nil {
+                continue
```

Comment:
This would ideally allocation failure here would result in a `(no filename: out of memory)` printout, or something, if it happens when printing out the stack trace. I haven't really done anything more here than make it compile, but it's something to think about.